Game/Levels/Sum/L02_Sum.lean

```diff
@@ -17,8 +17,8 @@ open BigOperators
 Statement (n : ℕ) : ∑ i : Fin n, ((i : ℕ) + 1) = n + (∑ i : Fin n, (i : ℕ)) := by
   Hint "**Du**: Hmm, wieder `simp`?
 
-  **Robo**: Nicht ganz. `simp` benützt nur Lemmas, die klar eine Vereinfachung darstellen.
-  Im Lean-Duden sind diese Lemmas mit `@[simp]` markiert.
+  **Robo**: Nicht ganz. `simp` benutzt nur Lemmata, die klar eine Vereinfachung darstellen.
```

Comment:
```suggestion
  **Robo**: Nicht ganz. `simp` benützt nur Lemmata, die klar eine Vereinfachung darstellen.
```

Das `ü` ist laut Duden glaubs auch korrekt :)

Reply:
Das stimmt -- musste ich gerade nachsehen. Wobei dort auch steht, dass es meist österr. oder schweizerr. Deutsch betrifft. Mir ist's einerlei also nehme ich diesen Vorschlag an.

Reply:
Also ich habe in der Vergangenheit jedes benützt in benutzt geändert. Wenn wir einheitlich auf benützt schwenken, wäre mir das auch recht, aber es sollte schon einheitlich sein. (Am besten dann auch in style guide vermerken.)